### PR TITLE
Feature/remove mixins to starter pack

### PR DESCRIPTION
**Pull request by ryanfitton-ap**
Remove mixins and place in Starter Pack

Ticket: https://webjobs.agepartnership.co.uk/app/tasks/15034642

Related component library PR: https://github.com/AgePartnership/oleg-component-library/pull/116

Checks:

- Sanity
- Review
- Malice

### CODE REVIEW

**Review by kimoshinowo98 — CHANGES_REQUESTED** [review]
Just one thing. I'm also not sure about whether it's ok for the lockfileVersion in package-lock.json to be updated - @beccadawson do you know if that's ok?

**Review by beccadawson — CHANGES_REQUESTED** [review]

File: base/_mixins.scss [hunk 1]
```diff
@@ -64,3 +66,160 @@ Ideally, all mixins should be declared here but in some cases, they've been decl
     position: absolute;
     width: 1px;
 }
+
+//Mixin to provide icon animation and position styles
+/*
+    Usage:
+        @include icon-setup('left', 'x', 0); //Icon positioned Left, animate on X axis, 0px margin for between text and icon
```
[the remark this answers is not in the document]
Comment (ryanfitton-ap) [reply base/_mixins.scss]: Updated

File: package-lock.json
[the remark this answers is not in the document]
Comment (beccadawson) [reply package-lock.json:1]: Thought so, we didn't get round to adding automatic switching stuff everywhere

Running `nvm use` should switch you onto the right versions

[the remark this answers is not in the document]
Comment (ryanfitton-ap) [reply package-lock.json:1]: @beccadawson 

Okay, tried these:


```
nvm use lts/erbium

nvm version

* States: v12.22.12

npm cache clear --force

* Deleted the node modules folder too

npm install
```

But no new changes have been tracked in the lockfile

[the remark this answers is not in the document]
Comment (beccadawson) [reply package-lock.json:1]: Please can you paste develop's lockfile over yours (or delete the file?) then run npm install (whilst on erbium)

[the remark this answers is not in the document]
Comment (ryanfitton-ap) [reply package-lock.json:1]: @beccadawson Still no luck. I tried early version before 'erbium', which was 'v12.14.1', this seemed to install but Travis checks did not pass.

**Comment by ryanfitton-ap** [comment]
@beccadawson @kimoshinowo98 Updated the readme with notes on the correct nvm and npm versions to use. 

The package.lock file has been updated too, with just one small change made when compared with the develop version.

**Comment by ryanfitton-ap** [comment]
@beccadawson  Updated the package .json to use version 1.2.0 of the Stylelint Config.

But now seeing different errors in Travis: https://app.travis-ci.com/github/AgePartnership/oleg-starter-pack/builds/262475440

**Comment by ryanfitton-ap** [comment]
@beccadawson  passing now, removed the git+ssh from the package dependency.

**Review by kimoshinowo98 — COMMENTED** [review]
Changes in package-lock now mostly look fine apart from:
- `node-releases` goes up by a major version
-  `yallist` and `lru-cache` are both going down by a major version

Worth installing this branch on the web.ap and mjolnir-templates repos and making sure Travis passes

**Review by kimoshinowo98 — APPROVED** [review]

**Pull request merged**